python/hail/methods/impex.py

```diff
@@ -548,7 +552,95 @@ def grep(regex, path, max_count=100):
            contig_recoding=nullable(dictof(strlike, strlike)))
 def import_bgen(path, tolerance=0.2, sample_file=None, min_partitions=None, reference_genome=None,
                 contig_recoding=None):
-    rg = reference_genome if reference_genome else Env.hc().default_reference
+    """Import BGEN file(s) as matrix table.
+
+    Warning
+    -------
+
+    A BGEN file must have a ``.idx`` file which can be generated by
+    :func:`.index_bgen`
+
+    Examples
+    --------
+
+    Import a BGEN file as a matrix table renaming contig name "01" to "1".
+
+    >>> ds_result = methods.import_bgen("data/example3.bgen",
+    ...                                 sample_file="data/example3.sample",
+    ...                                 contig_recoding={"01": "1"})
+
+    Notes
+    -----
+
+    Hail supports importing data in the BGEN file format. For more information
+    on the BGEN file format, see `here
+    <http://www.well.ox.ac.uk/~gav/bgen_format/bgen_format.html>`__. Note that
+    only v1.1 and v1.2 BGEN files are supported at this time. For v1.2 files,
+    only **unphased** and **diploid** genotype probabilities are allowed and the
+    genotype probability blocks must be either compressed with zlib or
+    uncompressed.
+
+    To load multiple files at the same time, use :ref:`Hadoop Glob Patterns
+    <sec-hadoop-glob>`.
+
+    **Column Fields**
+
+    - `s` (:class:`.TString`) -- Column key. This is the sample ID imported
```

Comment:
Most places in the docs I've seen, these bullet lists of fields are formatted with the field name bold: `*s*`. I don't think we've agreed on a style yet though.